src/components/WorldWeatherPage/WorldWeatherPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useEffect } from 'react';
+
+import weatherApi from '../../api/weatherApi';
+import { WorldWeatherContext } from '../../core/contexts';
+import { getCookie } from '../../utils/cookies';
+import Preloader from '../Preloader/Preloader';
+import WorldWeatherItem from '../WorldWeatherItem/WorldWeatherItem';
+
+import './WorldWeatherPage.scss';
+
+const locationDisplayedCities = [
+  '-0.118092,51.509865', /* London */
+  '-118.243683,34.052235', /* Los Angeles */
+  '37.618423,55.751244', /* Moscow */
+  '13.404954,52.520008', /* Berlin */
+  '121.469170,31.224361', /* Shanghai */
+  '116.383331,39.916668', /* Beijing */
+  /* '72.877426,19.076090', Mumbai
+  '28.979530,41.015137', Istanbul
+  '139.839478,35.652832', Tokyo
+  '-73.935242,40.730610', New York
+  '2.349014,48.864716', Paris */
+];
+
+function WorldWeatherPage() {
+  const [worldWeather, setWorldWeather] = useContext(WorldWeatherContext);
+
+  // using setTimeout to bypass 429 error from server (Too Many Requests);
+  useEffect(() => {
+    if (!worldWeather.length) {
+      const token = getCookie('token');
+      locationDisplayedCities.forEach((cityLocation, index) => {
+        setTimeout(() => {
+          weatherApi.getLocationInfo(cityLocation, token)
+            .then((data) => {
+              weatherApi.getCurrentWeather(cityLocation, token)
+                .then((weather) => {
+                  const fullData = { ...data, ...weather };
+                  setWorldWeather((prev) => ([...prev, fullData]));
+                });
+            });
+        }, (index + 1) * 250);
```

Comment:
if server returns you 429 error, please don't make it suffer. also your user won't be happy if network speed slows down and he needs to wait for responses from your synced requests

Reply:
In order for the server not to suffer, I need a non-existent request. Either the server developer must make the necessary rest api, or another technical task. It is impossible to accomplish the task in any other way.

Reply:
server developer doesn't have requirements to make for you some special request. but you have API with requests that provides necessary data and you don't have a requirement to show all that data in one moment on the same page. so task is feasible but you need to think about another solution